settings.gradle.kts

```diff
@@ -17,13 +17,15 @@
  *
  *
  */
+enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
I think this is not necessary from Gradle 7.4. Do you think we can remove it ? 
People reported that this was removed actually on Gradle 8, making it to fail when that eventual upgrade will happen 😅

Reply:
I know it fails on Gradle 8.0 (already tried to upgrade to Gradle 8.0, but requires newer AGP which isn't compatible with a stable version of IntelliJ yet).

Didn't know it was already stable tho. Good to know! I'll give it a shot